docker/upgrade.pp

```diff
@@ -0,0 +1,11 @@
+node default {
+  class { '::puppet_agent':
+    package_version => $facts['to_version'],
+    # Upgrades in Docker cannot start the puppet service due to systemd
```

Comment:
In the past this has been poorly documented, but `collection` is a required parameter when performing upgrades in FOSS scenarios.

We have hopefully updated the module's README enough to reflect this now.

```
    collection => $facts['to_collection'],
```